app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,10 @@
+Total Quotes Found: <%= @total_count %>
+
+<% @quotes.each do |quote| %>
+    <ul class='quote' id="id-<%= quote.id %>">
+      <li><%= quote.author %></li>
+      <li><%= quote.content %></li>
+      <li><%= quote.tags %></li>
+    </ul>
+  <br>
+<% end %>
```

Comment:
👍 The only thing you're missing here is the users keyword that they searched for.

Reply:
Yeahhh..woops